packages/launch-editor/guess.js

```diff
@@ -24,16 +24,32 @@ module.exports = function guessEditor (specifiedEditor) {
   try {
     if (process.platform === 'darwin') {
       const output = childProcess
-        .execSync('ps x', {
+        .execSync('ps x -o comm=', {
```

Comment:
This changes the output to only contain the process path and not any additional information like process ids.